pkg/security/secl/validators/rule_structure_test.go

```diff
@@ -91,14 +92,14 @@ func TestHasBareWildcardInField(t *testing.T) {
 		{
 			name: "not always true chained",
 			args: args{
-				ruleExpression: "exec.file.path =~ \"/**\" && exec.file.name != \"ls\" || open.file.name == \"myfile.txt\"",
+				ruleExpression: "exec.file.path =~ \"/**\" && exec.file.name != \"ls\" || exec.file.name == \"myfile.txt\"",
```

Comment:
Do you think you could keep one of those to test that we actually report an error if multiple event types are defined at the same time ?

Reply:
Yes that is a good idea, I'll add a test for that